test/guest/fedora-atomic.setup

```diff
@@ -18,12 +18,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+# Use mac's in name policy, prevents interference from network-scripts
```

Comment:
My objection is that I'd rather make the test more robust than tweak the OS to make a fragile test happy.  Thus, we should probably explicitly create two settings for the same mac, activate one, and check that Cockpit shows the correct one.

However, since this means we also need to figure out the metrics issue, it's probably better to do all this in another PR.